skins/oasis/modules/templates/PageHeader_Index.php

```diff
@@ -7,12 +7,20 @@
 	}
 }
 
+if ( $showCuratedContentToolBtn ) {
+	$curatedContentToolButton = '<a id="CuratedContentTool" class="wikia-button">' . wfMsg( 'wikiacuratedcontent-edit-mobile-main-page' ) . '</a>';
```

Comment:
Hard to tell. It's good to remove deprecations and vulnerabilities, but on the other hand those possible regressions… I would still do it, but first ask someone from platform (ping @macbre, @michalroszka) is should we do it and should we use here the `->escaped()` method instead of `->text()`.